openslides/motions/signals.py

```diff
@@ -189,13 +189,16 @@ def create_builtin_workflows(sender, **kwargs):
                                      allow_submitter_edit=True)
     state_1_2 = State.objects.create(name=ugettext_noop('accepted'),
                                      workflow=workflow_1,
-                                     action_word=ugettext_noop('Accept'))
+                                     action_word=ugettext_noop('Accept'),
+                                     css_class='success')
     state_1_3 = State.objects.create(name=ugettext_noop('rejected'),
                                      workflow=workflow_1,
-                                     action_word=ugettext_noop('Reject'))
+                                     action_word=ugettext_noop('Reject'),
+                                     css_class='danger')
     state_1_4 = State.objects.create(name=ugettext_noop('not decided'),
                                      workflow=workflow_1,
-                                     action_word=ugettext_noop('Do not decide'))
+                                     action_word=ugettext_noop('Do not decide'),
+                                     css_class='default')
```

Comment:
Not changed because new default (defined in model and not in template) is now `primary`. see above.